esaco-app/src/main/java/it/infn/mw/esaco/EsacoConfiguration.java

```diff
@@ -41,21 +45,36 @@
 
 import com.fasterxml.jackson.databind.ser.impl.SimpleFilterProvider;
 
+import eu.emi.security.authn.x509.ProxySupport;
 import eu.emi.security.authn.x509.X509CertChainValidatorExt;
+import eu.emi.security.authn.x509.impl.CertificateUtils.Encoding;
+import eu.emi.security.authn.x509.impl.DirectoryCertChainValidator;
+import eu.emi.security.authn.x509.impl.RevocationParametersExt;
 import eu.emi.security.authn.x509.impl.SocketFactoryCreator;
+import eu.emi.security.authn.x509.impl.ValidatorParamsExt;
 import it.infn.mw.esaco.exception.SSLContextInitializationError;
 import it.infn.mw.esaco.service.TimeProvider;
 import it.infn.mw.esaco.service.impl.IamDynamicServerConfigurationService;
 import it.infn.mw.esaco.service.impl.SystemTimeProvider;
+import it.infn.mw.esaco.util.x509.X509BlindTrustManager;
 
 @Configuration
 @EnableAutoConfiguration
 @EnableConfigurationProperties
 public class EsacoConfiguration {
 
+  public enum TrustAnchorsType {DIR, BUNDLE, NONE}
+  private static final int TRUST_ANCHORS_BUNDLE_CONNECTION_TIMEOUT_CA = 5000;
```

Comment:
Is this in msec? if so I would call it `TRUST_ANCHORS_BUNDLE_CONNECTION_TIMEOUT_CA_MSEC`
It would also be good to have this configurable, i.e. `x509.trustAnchorsBundle.connectionTimeoutMsec`

Reply:
It sets default value for https://github.com/eu-emi/canl-java/blob/master/src/main/java/eu/emi/security/authn/x509/impl/DirectoryCertChainValidator.java#L54, other than `0` (infinite). I did not make it configurable since I have not expected to make use of it in any practical way. As far as I understand it, is sets timeout for downloading CA bundles via HTTP(S). Right now, we have support only for `DIR`, `BUNDLE`, or `NONE` in `TrustAnchorsType`. Admittedly, keeping it at `0` would make much more sense ... `5000` looks like a truly random constant.

Reply:
Should I change it to `0` or make it fully configurable?

Reply:
(and, yes, I can add MSEC to the name as well, of course)